tests/tests/test_timetable.py

```diff
@@ -85,11 +88,45 @@ def test_conflicts(
         }
     ]
     requests.post(f"{EDITOAST_URL}/timetable/{timetable_id}/train_schedule", json=train_schedule_payload)
-    response = requests.get(f"{EDITOAST_URL}/timetable/{timetable_id}/conflicts/?infra_id={small_infra.id}")
-    assert response.status_code == 200
-    actual_conflicts = {conflict["conflict_type"] for conflict in response.json()}
+
+    conflicts_response = requests.get(f"{EDITOAST_URL}/timetable/{timetable_id}/conflicts/?infra_id={small_infra.id}")
+    assert conflicts_response.status_code == 200
+    actual_conflicts = {conflict["conflict_type"] for conflict in conflicts_response.json()}
     assert actual_conflicts == expected_conflict_types
 
+    # Check GET reservation block starts at the right time for the signal protecting switch.
+    # Train is received on closed (STOP/SHORT_SLIP_STOP) or OPEN signal.
+    # The free-block requirement must start at the same time as the spacing requirement of the switch's zone
+    # (signal sight for OPEN reception, or 20s before restart for STOP/SHORT_SLIP_STOP reception).
+    train_id = stopping_train_schedule_response.json()[0]["id"]
+    simu_response = requests.get(
+        f"{EDITOAST_URL}/train_schedule/{train_id}/simulation/?infra_id={small_infra.id}"
+    ).json()
```

Comment:
nit: you may want to check statuses

Reply:
Thanks, amended the 4th commit and pushed-force directly https://github.com/OpenRailAssociation/osrd/compare/461433f4d9aa532fc76cb4eb04dffa6d1b7d0ba5..34ba402dfbc926c3a485ff1b0d2a2431c1271daf